tests/test_transforms/test_feature_selection/test_feature_importance_transform.py

```diff
@@ -269,3 +272,84 @@ def test_mrmr_right_regressors(relevance_table, ts_with_regressors):
         if column.startswith("regressor"):
             selected_regressors.add(column)
     assert set(selected_regressors) == {"regressor_useful_0", "regressor_useful_1", "regressor_useful_2"}
+
+
+@pytest.mark.parametrize(
+    "transform, inversed_transform",
+    [
+        [
+            TreeFeatureSelectionTransform(
+                model=DecisionTreeRegressor(random_state=42), top_k=3, features_to_use="all", return_features=False
+            ),
+            TreeFeatureSelectionTransform(
+                model=DecisionTreeRegressor(random_state=42), top_k=3, features_to_use="all", return_features=True
+            ),
+        ],
+        [
+            MRMRFeatureSelectionTransform(relevance_table=StatisticsRelevanceTable(), top_k=3),
+            MRMRFeatureSelectionTransform(relevance_table=StatisticsRelevanceTable(), top_k=3, return_features=True),
+        ],
+    ],
+)
+def test_feature_importance_inverse_transform_save_columns(ts_with_regressors, transform, inversed_transform):
```

Comment:
You only change the logic in the base class, therefore you can write the tests in the file dedicated to the base class, just use any instance of it's children like [here](https://github.com/tinkoff-ai/etna/blob/45249270a8922acdbeeadee7e05844d5821a2ebc/tests/test_transforms/test_feature_selection/test_base_feature_selection_transform.py#L41)

Reply:
Also the logic of the test is a bit strange) 
I guess we need something like:
- As in the link replace the `selected_features` wit the predefined
- Call `transform` and check that the columns are saved correctly